pages/osx/carthage.md

```diff
@@ -0,0 +1,23 @@
+# carthage
+
+> Carthage is a dependency management tool for Swift and Objective-C.
+
+- Update the current set of dependencies:
```

Comment:
We should also mention that it also *fetches* and *builds* the dependencies from the Cartfile. Its not clear how the tools gets the dependencies in the first place.

How about this - 
```
Download and build all dependencies mentioned in Cartfile. Also used to update dependencies to their latest version.
```